packages/wonder-blocks-core/src/hooks/use-latest-ref.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+
+/**
+ * The useLatestRef hook returns a ref that always contains the `value` passed
+ * to the hook during the most recent render.
+ *
+ * It can be used to wrap a possibly-changing prop in a stable value that can
+ * be passed to useEffect without causing unnecessary re-renders. See the
+ * Storybook docs for a usage example.
```

Comment:
**nit:** We use the imperative and a single line for the first part of a function description.
```suggestion
 * Return a ref that always contains the `value` passed.
 *
 * The useLatestRef hook returns a ref that always contains the `value` passed
 * to the hook during the most recent render.
 *
 * It can be used to wrap a possibly-changing prop in a stable value that can
 * be passed to useEffect without causing unnecessary re-renders. See the
 * Storybook docs for a usage example.
```